### PR TITLE
IW-1341 | Close navigation when sub-level title is clicked

### PR DESCRIPTION
**Pull request by mszabo-wikia**
If an user clicks a level 2 item title in mobile-wiki community header, the navigation bubble remains visible even though the page has changed. The navigation should always be closed if it triggers a page change.

https://wikia-inc.atlassian.net/browse/IW-1341

@Wikia/iwing

### CODE REVIEW

**Comment by mszabo-wikia** [comment]
I only just noticed https://github.com/Wikia/design-system/pull/316, probably worth combining the two

File: app/components/community-bar/local-navigation-dropdown.js [hunk 1]
```diff
@@ -78,6 +78,11 @@ export default Component.extend({
 		this.onLinkClicked(event);
 	},
 
+	onSubLevelTitleClicked(event) {
```
Comment (hakubo) [inline app/components/community-bar/local-navigation-dropdown.js]: hmm this looks exactly the same as onThirdLevelSelected do we need another method?

Comment (hakubo) [reply app/components/community-bar/local-navigation-dropdown.js]: maybe rename the method so it could apply to more places?

**Comment by hakubo** [comment]
@mszabo-wikia feel free to merge this https://github.com/Wikia/design-system/pull/316 to this PR.

**Pull request merged**